src/libraries/Microsoft.PowerFx.Core/Public/Engine.cs

```diff
@@ -68,7 +68,7 @@ public IEnumerable<string> GetAllFunctionNames()
         // Additional symbols for the engine.
         // A derived engine can replace this completely to inject engine-specific virtuals. 
         // These symbols then feed into the resolver
-        protected SymbolTable EngineSymbols { get; set; }
+        protected ReadOnlySymbolTable EngineSymbols { get; set; }
```

Comment:
This is technically breaking change (and is flagged by Bruce's tool).  Least privilege - we should be demoting the type. 
Does not impact PowerApps.